app/src/main/java/com/ciechu/kursakademiaandroida/core/di/AppModule.kt

```diff
@@ -0,0 +1,20 @@
+package com.ciechu.kursakademiaandroida.core.di
+
+import androidx.recyclerview.widget.DividerItemDecoration
+import androidx.recyclerview.widget.GridLayoutManager
+import androidx.recyclerview.widget.LinearLayoutManager
+import org.koin.android.ext.koin.androidContext
+import org.koin.dsl.module
+
+val appModule = module {
+
+    single {
```

Comment:
Te elementy widoku dobrze będzie zdefiniować jako single. Gdy nie będą używane, to szybciej zostaną wyczyszczone z pamięci.